src/opm/input/eclipse/Schedule/BCProp.cpp

```diff
@@ -87,6 +103,38 @@ BCProp::BCFace::BCFace(const DeckRecord& record) :
     if (const auto& T = record.getItem<BCKEY::TEMPERATURE>(); ! T.defaultApplied(0)) {
         temperature = T.getSIDouble(0);
     }
+    MechBCValue mechbcvaluetmp;
+    if (const auto& P = record.getItem<BCKEY::STRESSXX>(); ! P.defaultApplied(0)) {
+        mechbcvaluetmp.stress[0] = P.getSIDouble(0);
+    }
+    if (const auto& P = record.getItem<BCKEY::STRESSYY>(); ! P.defaultApplied(0)) {
+        mechbcvaluetmp.stress[1] = P.getSIDouble(0);
+    }
+    if (const auto& P = record.getItem<BCKEY::STRESSZZ>(); ! P.defaultApplied(0)) {
+        mechbcvaluetmp.stress[2] = P.getSIDouble(0);
+    }
+    mechbcvaluetmp.stress[3] = 0;
```

Comment:
Why do we keep the three last numbers of `stress` if they are always set to zero?

Reply:
I could have added stressxy also as value, but have not so fare. I thought it was better to make a proper stress i.e. 6 values xx,yy,zz,yz,xz,xy even if input did not support it.